cloud-stream-kafka/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot'
+	id 'org.springframework.aot.smoke-test'
+	id 'org.graalvm.buildtools.native'
+}
+
+ext {
+	set('springBootVersion', "3.0.0-M4")
```

Comment:
Why don't we use current boot snapshots like the other smoke tests do?

Reply:
It is because SCSt is on SB-M4 and there are breaking changes in SNAPSHOT that we can't make in SCSt until it bumps up ([SCSt #2465](https://github.com/spring-cloud/spring-cloud-stream/issues/2465)). I will follow up w/ Oleg to see if there is anything blocking that bump to SNAPSHOT in SCSt.

Reply:
The idea with the smoketests was that they provide a picture what users can expect when they run native-image with the current SB 3.0 SNAPSHOTs. That's why I'm hesitant with changes that meddle with dependency versions.

Reply:
Let's see what Oleg is saying.

Reply:
Agreed. I also have a [proposal](https://github.com/spring-cloud/spring-cloud-stream/pull/2474) queued up in SCSt to address this.

Reply:
The above PR has been merged so I will update this to use snapshots sometime today.

Reply:
Back to SB 3.0.0-SNAPSHOT @mhalbritter 😸